model_manager/src/transformers/BaseTransformer.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+class BaseTransformer:
```

Comment:
Type hinting for the config as well just to clarify that the object passed is a dictionary and not string to the .yml dictionary.